.github/workflows/staticcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: "staticcheck"
+on: ["push", "pull_request"]
+
+jobs:
+  ci:
+    name: "Run Staticcheck"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - uses: WillAbides/setup-go-faster@v1.7.0
+      with:
+        go-version: 1.19.x
+    # - run: "go vet ./..."
```

Comment:
If this is being made part of the same workflow eventually, we should call this job `"Run Linters"` and rename this workflow and file. Maybe it should just be a separate workflow?

Reply:
Also, we can reenable this and just set `continue-on-error: true`: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error